### ISSUE TITLE
Automatic error report

### ISSUE THREAD

**Issue by nrkarp** [spyder-ide/spyder#7899]
I'm attempting to connect to a remote iPython kernel on paperspace.com, using Spyder 3.28 on Mac OS X 10.13.6

Followed recipe at https://github.com/ipython/ipython/wiki/Cookbook:-Connecting-to-a-remote-kernel-via-ssh

Using that recipe I can open a Qtconsole using the existing kernel successfully, but only from command line. Fails within Spyder.

Steps to reproduce the Spyder error:
1) On remote server: 
	ipython kernel
=> generates 
	kernel-5465.json

2) In local terminal:
	scp paperspace@74.82.31.199:/run/user/1000/jupyter/kernel-5465.json ./
	(this successfully transfers the .json file)

3) In Spyder (console gear->Connect to an existing terminal)
	Connection info: /Users/nrk/kernel-5465.json
	Check "This is a remote kernel"
	Host name: paperspace@74.82.31.199
	Ssh key: ~/.ssh/id_rsa

4) Fails to connect, with indicated error report.

5) After failure, successfully opened a QtConsole from terminal with:
    ipython qtconsole --existing ./kernel-5465.json --ssh paperspace@74.82.31.199

Two tangential items:
*On Mac, folders with leading '.' are not visible in file selection dialog. e.g. the folder containing the rsa key. The full ssh key path was entered manually, as above.
*My RSA key was generated without the '.pem' suffix that the Spyder dialog suggests. I tried the above sequence both without the suffix ('rsa_id') and renaming to include the suffix ('rsa_id.pem') 

### Traceback
```python-traceback
  File "/anaconda3/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 1060, in create_client_for_kernel
    password)
  File "/anaconda3/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 1782, in _create_client_for_kernel
    client.shellwidget.kernel_client = kernel_client
  File "/anaconda3/lib/python3.6/site-packages/qtconsole/base_frontend_mixin.py", line 57, in kernel_client
    kernel_client.iopub_channel.message_received.connect(self._dispatch)
AttributeError: 'QtKernelClient' object has no attribute 'iopub_channel'
```

## Package Versions

* Spyder version: 3.2.8 
* Python version: 3.6.5
* Qt version: 5.9.4
* PyQt5 version: 5.9.2
* Operating system: Darwin 17.7.0

### Dependencies

```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.4.0 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.23.0 (OK)
numpy >=1.7      :  1.14.3 (OK)
sphinx >=0.6.6   :  1.7.4 (OK)
rope >=0.9.4     :  0.10.7 (OK)
jedi >=0.9.0     :  0.12.0 (OK)
psutil >=0.3     :  5.4.5 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.28.2 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.4.0 (OK)
pylint >=0.25    :  1.8.4 (OK)
```

**Comment by ccordoba12** [comment]
> Followed recipe at https://github.com/ipython/ipython/wiki/Cookbook:-Connecting-to-a-remote-kernel-via-ssh

This is not the right way to do it with Spyder. Please see our docs for this:

http://docs.spyder-ide.org/ipythonconsole.html#connect-to-an-external-kernel

>  File "/anaconda3/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 1060, in create_client_for_kernel
    password)
  File "/anaconda3/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 1782, in _create_client_for_kernel
    client.shellwidget.kernel_client = kernel_client
  File "/anaconda3/lib/python3.6/site-packages/qtconsole/base_frontend_mixin.py", line 57, in kernel_client
    kernel_client.iopub_channel.message_received.connect(self._dispatch)
AttributeError: 'QtKernelClient' object has no attribute 'iopub_channel'

This error was fixed in our 3.3.1 version, so please update.

**Comment by ccordoba12** [comment]
This is a duplicate of issue #7082 and it was fixed in Spyder 3.3.1, so please update to our latest version (3.3.4) to get the fix.
